libfwupdplugin/fu-udev-device.h

```diff
@@ -112,10 +62,6 @@ void
 fu_udev_device_set_bind_id(FuUdevDevice *self, const gchar *bind_id) G_GNUC_NON_NULL(1);
 const gchar *
 fu_udev_device_get_driver(FuUdevDevice *self) G_GNUC_NON_NULL(1);
-gboolean
-fu_udev_device_is_pci_base_cls(FuUdevDevice *self, FuPciBaseCls cls) G_GNUC_NON_NULL(1);
-guint32
-fu_udev_device_get_cls(FuUdevDevice *self) G_GNUC_NON_NULL(1);
```

Comment:
these should probably be _mentioned_ in migration guide

Reply:
Good idea, fixed.